samples/client/src/main/java/global/covesa/sdk/client/ui/PushUiState.kt

```diff
@@ -6,9 +6,17 @@ import global.covesa.sdk.api.client.push.PushManager
 data class PushUiState(
     val registered: Boolean = false,
     val pushDistributor: String = "",
+    val availableDistributors: List<String> = emptyList(),
+    val selectedDistributor: String = "",
+    val savedDistributor: String = "",
```

Comment:
If we resume:
- pushDistributor is the saved distrib + ack
- selectedDistributor is initially the saved distributor, then the new selected when the user select another distrib
- savedDistributor is the saved distrib

I think we should move them to `String?`, what to you think ?